### DISCUSSION TITLE
우아한테크코스 회고

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#92] [General]
# 우아한테크코스 회고

길고 긴 10개월이 지나고, 벌써 레벨4가 마무리되는 지금 우아한테크코스를 회고해보려 합니다. 베로 위키 애독자들도 모를 수도 있는 올해 2월 6일 회고를 가져와 보겠습니다. 2월 6일이면 첫 등교 전 날이죠. 사실 저는 이때 마음의 병이 있어 상담을 받고 있었는데요, 지칠대로 지쳐 제가 뭘 할 수 있을지에 대해 의심만 가득한 상태였습니다. 안일한 마음가…

https://vero.wiki/woowacourse-retrospect/

**Comment by carsago** [comment]
저도 전에는 병원도 다니고 자기비하나 항상 남들에게 뒤쳐지면 안되는 압박감도 심했는데 이제는 그렇지 않네요.

베로도 프로그래밍 이전에 인간적으로 성장한 것 같아 멋지네요.

**Reply by Cyma-s** [reply]
맞아요 여기와서 그게 많이 나은 것 같아서 항상 감사합니다
오리도 행복하세요

**Comment by Creative-Lee** [comment]
은솔아 너는 멋있는 사람이야

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋ 도현 씨도 멋진 사람이잖아요^^ 행복하세요
